scoop/src/main/java/com/lyft/scoop/Router.java

```diff
@@ -24,7 +27,9 @@ public boolean goBack() {
                 Scoop nextScoop = backStack.peek();
                 Screen nextScreen = Screen.fromScoop(nextScoop);
                 performScoopChange(nextScoop, nextScreen, previousScreen, TransitionDirection.EXIT);
-
+                return true;
+            } else if (hasEmptyStack) {
```

Comment:
@lexer Added one to disable back button in a Dialog, then dismiss it. Back button still works fine.